priv/templates/extended_bin

```diff
@@ -343,6 +343,24 @@ case "$1" in
              "unpack" "$REL_NAME" "$NAME_TYPE" "$NAME" "$COOKIE" "$2"
         ;;
 
+    uninstall)
```

Comment:
once it's merged into master i'll let you know so you can rebase and apply your changes